File: backend/templates/terraform/lambda_placeholder.py
"""
Placeholder Lambda Function
This will be replaced by actual application code during deployment
"""

import json
import logging

logger = logging.getLogger()
logger.setLevel(logging.INFO)


def handler(event, context):
    """Lambda handler function"""
    logger.info('Event: %s', json.dumps(event))

    return {
        'statusCode': 200,
        'headers': {
            'Content-Type': 'application/json',
            'Access-Control-Allow-Origin': '*'
        },
        'body': json.dumps({
            'message': 'Lambda function deployed successfully. Please update with your application code.',
            'event': event
        })
    }

File: backend/templates/terraform/lambda_placeholder.js
// Placeholder Lambda Function
// This will be replaced by actual application code during deployment

exports.handler = async (event) => {
    console.log('Event:', JSON.stringify(event, null, 2));

    return {
        statusCode: 200,
        headers: {
            'Content-Type': 'application/json',
            'Access-Control-Allow-Origin': '*'
        },
        body: JSON.stringify({
            message: 'Lambda function deployed successfully. Please update with your application code.',
            event: event
        })
    };
};
